pkg/storage/sds/edv/formatprovider/formatprovider.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package formatprovider
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/storage/sds"
+	"github.com/hyperledger/aries-framework-go/pkg/storage/sds/edv/documentprocessor"
```

Comment:
we might need to come back to this topic - sds/edv seems a bit odd, but I don't currently have a better suggestion.

Reply:
Removed the `sds` folder and just moved `edv` to be directly under `storage` for now since everything that's in there is purely EDV-related. The SDS spec is still in flux so perhaps this should change in the future.